internal/developer/schema.sql

```diff
@@ -44,3 +44,17 @@ ALTER TABLE developer_profile ADD COLUMN search_status VARCHAR(20) NOT NULL DEFA
 ALTER TABLE developer_profile ADD COLUMN role_types VARCHAR(20) NOT NULL DEFAULT 'full-time';
 ALTER TABLE developer_profile ADD COLUMN detected_location_id VARCHAR(255) DEFAULT NULL;
 ALTER TABLE developer_profile_message ADD COLUMN sender_id CHAR(27) DEFAULT NULL;
+
+
+CREATE TYPE valid_developer_metadata_type AS ENUM ('experience', 'education', 'github');
```

Comment:
these schema files under /internal are obsolete, I will open a ticket for this. Can you please copy paste this into the schema sql and add some fixtures at the root of the repo please? so it's easier for local development

Reply:
Moved.